src/components/CurrentWeather/CurrentWeather.jsx

```diff
@@ -0,0 +1,43 @@
+import React from 'react'
+import "./CurrentWeather.scss"
+// Importing Image
+import WeatherImages from '../../WeatherImages/WeatherImages'
+
+const CurrentWeather = ({weatherInfo}) => {
+
+  if(weatherInfo.length > 0){
+    
+  const infWeather =  weatherInfo[0].weather[0].id 
+ 
+  // const WeatherIcons = (infWeather)=>{
+  //   return (
+  //   infWeather < 300 ? <img src={Storm} />
+  //   :  infWeather >= 300 && infWeather <= 499 ? <img 
+  //   src={Drizzle} /> : infWeather >= 500 && infWeather <= 599 ? <img 
+  //   src={Rain} /> : infWeather >= 600 && infWeather <= 699 ? <img 
+  //   src={Snow} /> : infWeather >= 700 && infWeather <= 799 ? <img 
+  //   src={Fog} /> : infWeather === 800 ? <img 
+  //   src={Clear} /> : infWeather === 801 ? <img 
+  //   src={PartlyCloudy} /> : infWeather >= 802 && infWeather <= 805 ? <img 
+  //   src={MostlyCloudy} /> :
+  //    <p>No Image Found</p> 
+  // )}
```

Comment:
It's best not to commit commented out code like this. I would delete it. 

Especially because your `WeatherImage` component is fulfilling this function on line 29 :)